unyt/unit_object.py

```diff
@@ -722,6 +696,84 @@ def latex_representation(self):
 #
 
 
+em_conversions = {
+    "C": ("esu", 0.1*speed_of_light_cm_per_s, "cgs"),
+    "T": ("gauss", 1.0e4, "cgs"),
+    "Wb": ("Mx", 1.0e8, "cgs"),
+    "A": ("statA", 0.1*speed_of_light_cm_per_s, "cgs"),
+    "V": ("statV", 1.0e-8*speed_of_light_cm_per_s, "cgs"),
+    "ohm": ("statohm", 1.0e9/(speed_of_light_cm_per_s**2), "cgs"),
+    "esu": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "Fr": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "statC": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "gauss": ("T", 1.0e-4, "mks"),
+    "G": ("T", 1.0e-4, "mks"),
+    "Mx": ("Wb", 1.0e-8, "mks"),
+    "statA": ("A", 10.0/speed_of_light_cm_per_s, "mks"),
+    "statV": ("V", 1.0e8/speed_of_light_cm_per_s, "mks"),
+    "statohm": ("ohm", speed_of_light_cm_per_s**2*1.0e-9, "mks"),
+}
+
+
+def _em_conversion(orig_units, conv_data, to_units=None, unit_system=None):
```

Comment:
It looks like you're trying to do the more complicated EM conversions here still. Is that wrong?

Reply:
I guess it's not enforced in the function, but `to_units` will only be `None` if the `unit_system` is also not `None`. This happens in the call to `_em_conversion` in `in_base` and `get_base_equivalent`.

Reply:
Can we get a docstring or some comments here to explain a bit the steps of this function? I know this is internal, but for future reference it would help.

Reply:
Done. Let me know if you have comments about the docstrings I added.